src/backend/utils/indexer.js

```diff
@@ -11,21 +11,29 @@ const type = 'post';
 
 /**
  * Indexes the text and id from a post
- * @param text from a post
- * @param postId same id used to store on redis the post object where the text is from
+ * @param post
+ * @param post.text from a post
+ * @param post.id same id used to store on redis the post object where the text is from
+ * @param post.title from a post
+ * @param post.published from a post
+ * @param post.author from a post
  */
-const indexPost = async (text, postId) => {
+// post.text, post.id, post.title, post.updated, post.feed.author
```

Comment:
Remove this

Reply:
Sorry, I thought I removed all of parts of test....